nexus/test-utils/src/resource_helpers.rs

```diff
@@ -500,7 +500,73 @@ pub async fn delete_disk(
     project_name: &str,
     disk_name: &str,
 ) {
-    let url = format!("/v1/disks/{}?project={}", disk_name, project_name,);
+    let url = format!("/v1/disks/{}?project={}", disk_name, project_name);
+    object_delete(client, &url).await
+}
+
+pub async fn delete_snapshot(
+    client: &ClientTestContext,
+    project_name: &str,
+    snapshot_name: &str,
+) {
+    let url =
+        format!("/v1/snapshots/{}?project={}", snapshot_name, project_name);
+    object_delete(client, &url).await
+}
+
+pub async fn create_alpine_project_image(
+    client: &ClientTestContext,
+    project_name: &str,
+    image_name: &str,
+) -> views::Image {
+    let images_url = format!("/v1/images?project={}", project_name);
+    object_create(
+        client,
+        &images_url,
+        &params::ImageCreate {
+            identity: IdentityMetadataCreateParams {
+                name: image_name.parse().unwrap(),
+                description: String::from(
+                    "you can boot any image, as long as it's alpine",
+                ),
+            },
+            source: params::ImageSource::YouCanBootAnythingAsLongAsItsAlpine,
```

Comment:
Is this still around?  I see it in a bunch of places still, so I don't know if it was external only that got removed and the internal side stuff is different?

Reply:
Yeah, I think it was only removed from the openapi json.